src/Agent.Worker/DiagnosticLogManager.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) Microsoft Corporation.
+﻿// Copyright (c) Microsoft Corporation.
```

Comment:
Looks like you've saved this file in a different encoding - UTF-8 with BOM. This file was UTF-8 before. Could you please revert this change?